docs/dcp2_operating_procedures.rst

```diff
@@ -0,0 +1,253 @@
+    by the DCP, move on to the next step. Don't merge the PR just yet.
+
+4)  Prepare a PR against the ``develop`` branch of the `metadata-schema`_
+    repository. It is OK to perform this step concurrently with the preceding
+    steps.
+
+5)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. Don't merge the PR just yet.
+
+    Note that the above steps can be performed for several independent schema
+    changes in parallel.
+   
+6)  Merge the `metadata-schema`_ repository PR into the ``develop`` branch and
+    from there into the ``staging`` branch.
+
+7)  Prepare a PR against the `Test metadata`_ in the `schema-test-data`_
+    repository. Don't merge the PR just yet.
+
+8)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. As part of the review, the Azul team
+    updates their unit tests to validate the updated test metadata. If a test
+    fails, they either improve the test or the team member reviewing the PR
+    requests the necessarty changes to the test metadata.
+
+9)  Prepare a PR against the schema repository, targetting the main branch.
+
+10) Initiate a `Pull request review`_ of your PR. The review should be quick
+    because reviewers will already have seen the changes while reviewing the PR
+    against the ``develop`` branch. Reviewers should not request any substantial
+    changes at this stage unless last minute considerations absolutely require
+    it. Once the PR has been approved by the DCP, move on to the next step.
+
+11) Merge all PRs. You may squash and/or rebase a PR before merging it but that
+    should not affect the diff between the base and head commits of the PR
+    branch, except for housekeeping like resolving merge conflicts or adjusting
+    the host name of schema URLs in PRs against the `schema-test-data`_
+    repository. No semantic changes may be introduced to a PR after it has been
+    approved by the DCP. All such housekeeping should be done on the PR branch
+    prior to merging it.
+
+
+Test metadata
+-------------
+
+Every change to the schemas in the `metadata-schema`_ repository must be
+accompanied by a matching change to the test metadata in the `schema-test-data`_
+repository. Both changes must be done as PRs following the `Pull request
+review`_  process. 
+
+The test metadata should excercise all graph shapes in actual use. 
+
+The schema references (``describedBy``) in all metadata documents in the main
+branch of `schema-test-data`_ repository must match the schemas on the
+`master` branch of the `metadata-schema`_ repository. Pull requests against the 
+
+
+Pull request review
+-------------------
+
+As a PR author
+
+1)  Announce the PR on the `#dcp2` channel on Slack, @-mentioning all
+    `Designated reviewers`_.
+
+2)  Request a review from all `Designated reviewers`_.
```

Comment:
We can't refer to two lists because they could be inconsistent. PA is only used in the schema repo and I think this document should be self-contained, especially when it comes to the very important list of reviewers.

Reply:
With self-containing in mind, agree that there is no use in listing how does the PR author identify the designated reviewers

Reply:
won't fix